src/main/java/com/myjeeva/digitalocean/pojo/Project.java

```diff
@@ -0,0 +1,125 @@
+package com.myjeeva.digitalocean.pojo;
+
+import com.google.gson.annotations.Expose;
+import com.myjeeva.digitalocean.common.Environment;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+/**
+ * Represents Project attributes of DigitalOcean.
+ *
+ * @author Thomas Crombez (thomasc@trikthom.com)
+ */
+public class Project extends Base {
+
+//  private static final long serialVersionUID = ??;
```

Comment:
@TrikThom This ID could be generated using IDE.

Reply:
Thanks, I needed to turn on a setting in IntelliJ.